### ISSUE TITLE
PyParsing and HTTPLib2 versioning issue

### ISSUE THREAD

**Issue by abadrinath947** [berkeley-dsep-infra/hubs#2950]
### Bug description

Here is a traceback when attempting to use `otter assign` due to mismatched versions of pyparsing and httplib2, where I suspect pyparsing was recently updated and a variable was renamed. There's an issue in httplib2 about this:

https://github.com/httplib2/httplib2/issues/207

```
Traceback (most recent call last):
  File "/opt/conda/bin/otter", line 8, in <module>
    sys.exit(cli())
  File "/opt/conda/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/opt/conda/lib/python3.9/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/conda/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/conda/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/otter/cli.py", line 43, in assign_cli
    return assign(*args, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/otter/assign/__init__.py", line 39, in main
    return v0_main(master, result, no_pdfs=no_pdfs, no_run_tests=no_run_tests, username=username, 
  File "/opt/conda/lib/python3.9/site-packages/otter/assign/v0/__init__.py", line 74, in main
    pc = PluginCollection(plugins, "", {})
  File "/opt/conda/lib/python3.9/site-packages/otter/plugins/__init__.py", line 71, in __init__
    self._plugins = self._load_plugins(self._plugin_config, submission_path, submission_metadata)
  File "/opt/conda/lib/python3.9/site-packages/otter/plugins/__init__.py", line 98, in _load_plugins
    module = importlib.import_module(module)
  File "/opt/conda/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/conda/lib/python3.9/site-packages/otter/plugins/builtin/__init__.py", line 5, in <module>
    from .gmail_notifications import GmailNotifications
  File "/opt/conda/lib/python3.9/site-packages/otter/plugins/builtin/gmail_notifications/__init__.py", line 15, in <module>
    from googleapiclient.discovery import build
  File "/opt/conda/lib/python3.9/site-packages/googleapiclient/discovery.py", line 42, in <module>
    import httplib2
  File "/opt/conda/lib/python3.9/site-packages/httplib2/__init__.py", line 52, in <module>
    from . import auth
  File "/opt/conda/lib/python3.9/site-packages/httplib2/auth.py", line 20, in <module>
    auth_param_name = token.copy().setName("auth-param-name").addParseAction(pp.downcaseTokens)
```

CC: @fperez 

### Environment & setup

- Hub: Data 100
- Language: Python

### How to reproduce

1. Go to the terminal.
2. Use `otter assign file.ipynb output/`. If otter isn't available, perhaps try `import httplib2`.
3. See error above.

**Comment by yuvipanda** [comment]
Looks like https://github.com/httplib2/httplib2/pull/209 will fix the issue in httplib2. You can temporarily test otter by running the following command in your server (via a notebook cell):

```
!pip install -U --force git+https://github.com/busunkim96/httplib2.git@pp-downcaseTokens
```

**Comment by temoto** [comment]
you can wait less than hour for release too

**Comment by temoto** [comment]
pyparsing v3 support is released in `httplib2>=0.20.2`

**Comment by yuvipanda** [comment]
THANK YOU SO MUCH, @temoto!!!!!
